l1-contracts/deploy-scripts/LensScript.s.sol

```diff
@@ -0,0 +1,371 @@
+
+        // proxy admin - erc20 bridge
+        tempEcoOwnershipCalls[3] = IGovernance.Call({
+            target: config.tempProxyAdmin,
+            value: 0,
+            data: abi.encodeCall(
+                ProxyAdmin.changeProxyAdmin,
+                (ITransparentUpgradeableProxy(config.tempErc20Bridge), config.newProxyAdmin)
+            )
+        });
+
+        // bridgehub - transfer ownership
+        tempEcoOwnershipCalls[4] = IGovernance.Call({
+            target: config.tempBridgehub,
+            value: 0,
+            data: abi.encodeCall(Ownable2Step.transferOwnership, (transitionaryOwner))
+        });
+
+        // state transition manager - transfer ownership
+        tempEcoOwnershipCalls[5] = IGovernance.Call({
+            target: config.tempStateTransitionManager,
+            value: 0,
+            data: abi.encodeCall(Ownable2Step.transferOwnership, (transitionaryOwner))
+        });
+
+        // validator timelock - transfer ownership
+        vtOwnershipCall = abi.encodeWithSelector(Ownable2Step.transferOwnership.selector, transitionaryOwner);
+
+        // IGovernance.Call({
+        //     target: config.tempValidatorTimelock,
+        //     value: 0,
+        //     data: abi.encodeCall(Ownable2Step.transferOwnership, (transitionaryOwner))
+        // });
+
+        // l1 shared bridge - transfer ownership
+        tempEcoOwnershipCalls[6] = IGovernance.Call({
+            target: config.tempL1SharedBridge,
+            value: 0,
+            data: abi.encodeCall(Ownable2Step.transferOwnership, (transitionaryOwner))
+        });
+
+        // transitionary owner Accept and transfer ownership - bh
+        tempEcoOwnershipCalls[7] = IGovernance.Call({
+            target: transitionaryOwner,
+            value: 0,
+            data: abi.encodeCall(TransitionaryOwner.claimOwnershipAndGiveToGovernance, (config.tempBridgehub))
+        });
+
+        // transitionary owner Accept and transfer ownership - stm
+        tempEcoOwnershipCalls[8] = IGovernance.Call({
+            target: transitionaryOwner,
+            value: 0,
+            data: abi.encodeCall(
+                TransitionaryOwner.claimOwnershipAndGiveToGovernance,
+                (config.tempStateTransitionManager)
+            )
+        });
+
+        // transitionary owner Accept and transfer ownership - vt
+        tempEcoOwnershipCalls[9] = IGovernance.Call({
+            target: transitionaryOwner,
+            value: 0,
+            data: abi.encodeCall(TransitionaryOwner.claimOwnershipAndGiveToGovernance, (config.tempValidatorTimelock))
+        });
+
+        // transitionary owner Accept and transfer ownership - l1 shared bridge
+        tempEcoOwnershipCalls[10] = IGovernance.Call({
+            target: transitionaryOwner,
+            value: 0,
+            data: abi.encodeCall(TransitionaryOwner.claimOwnershipAndGiveToGovernance, (config.tempL1SharedBridge))
+        });
+
+        tempEcoScheduleOperation = abi.encodeWithSelector(
```

Comment:
it is correct, but better to use `abi.encodeCall` just in case since it would give type validation